lib/task.js

```diff
@@ -149,6 +150,22 @@ function factory(
             });
     };
 
+    Task.prototype.getPath = function(obj, path) {
```

Comment:
@WangWinson you caught me. I had actually moved this in because my initial iteration of this change had everything in on-core instead of on-tasks, and I guess I overlooked getting rid of it. I'll switch to `_.get`.